flycheck-elsa.el

```diff
@@ -46,28 +46,119 @@
   :group 'flycheck-elsa
   :type '(repeat regexp))
 
-(defun flycheck-elsa--enable-p ()
-  "Return non-nil if we can enable Elsa in current buffer.
+(defcustom flycheck-elsa-command 'cask
```

Comment:
Do we expect people to configure this with dir-local variables?  I think that makes sense right?  Can we mention it in readme somewhere?

Reply:
> Do we expect people to configure this with dir-local variables?

this could indeed happen and it should work. I'm not sure why people would like to use the cask solution if Elsa is installed with its dependencies on the host though.

Reply:
Fair enough.

OK, I think this is good from my side, so if you have nothing else to add/change feel free to go ahead and merge.